spec/debug/driver.cr

```diff
@@ -13,7 +13,7 @@ input = ARGV[0]
 bin = File.join(tmp_build_dir, "debug_test_case")
 debugger_script = File.join(tmp_build_dir, "./debugger.script")
 
-`#{repo_base_dir}/bin/crystal build --debug #{input} -o #{bin}`
+Process.run(ENV["CRYSTAL_SPEC_COMPILER_BIN"]? || "#{repo_base_dir}/bin/crystal", ["build", "--debug", input, "-o", bin])
```

Comment:
This was already problematic before, so feel free to ignore, but we now have a good opportunity to check that this command actually succeeded.